deps/build.jl

```diff
@@ -1,14 +1,19 @@
 using Compat
-import BinDeps: download_cmd, unpack_cmd, splittarpath
-if is_apple()
+using Compat: @info
```

Comment:
this needs a much newer compat minimum

Reply:
It needs at least Compat 0.47.0 to be precise.